cdc/processor/sinkmanager/redo_log_worker.go

```diff
@@ -116,8 +116,9 @@ func (w *redoWorker) handleTask(ctx context.Context, task *redoTask) (finalErr e
 						zap.Uint64("memory", refundMem))
 				}
 			}
+			sendRows := rows[0:]
```

Comment:
They still shares the same underlaying memory, I am not sure if it fixes the race.